nixui/parser/ast.py

```diff
@@ -100,9 +100,13 @@ def visit_node(self, node, visited_children):
     def visit_token(self, node, visited_children):
         """ Returns the overall output. """
         name, _, quoted, _, _, position, _ = visited_children
-        quoted = quoted[1:-1]  # strip quote symbols (TODO: move this to the grammar)
-        quoted = quoted.replace('EQ', '\"')  # TODO: fix hack (see note 3929 above)
-        quoted = quoted.encode('utf-8').decode('unicode_escape')  # unescape
+        quoted = quoted\
```

Comment:
Can pythons standard library perform this multi-line operation in a single line?

Reply:
TIL this exists: https://docs.python.org/3/library/stdtypes.html#str.translate

I'll replace it with that.

Reply:
This was a dumb from me lol. `translate` is a translation from character to character(or character to string). What we need is a translation from string to character.